src/views/Layout.jsx

```diff
@@ -1,18 +1,55 @@
-import { Outlet, NavLink } from 'react-router-dom';
+import { Outlet } from 'react-router-dom';
+import { useLocation } from 'react-router-dom';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import React, { useState } from 'react';
+import { Image } from 'react-bootstrap';
+import toast, { Toaster } from 'react-hot-toast';
 
 import './Layout.css';
 
-export function Layout() {
-	return (
-		<>
-			<div className="Layout">
-				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
+export function Layout({ listToken }) {
+	// useLocation hook will be used to check current route
+	const location = useLocation();
+
+	// Notify when adding item is unsuccessful
```

Comment:
Is this comment accurate? Should this be a successful toast?

Reply:
Fixed, thank you!